blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/OrganizationFactoryImpl.java

```diff
@@ -22,6 +24,10 @@ public class OrganizationFactoryImpl extends OrganizationFactory {
     private static final String ORGANIZATION_NAME = StringUtils.defaultIfBlank(
             System.getProperty("BLUE_ORGANIZATION_NAME"),"jenkins");
 
+    private static final String ROOT_FOLDER_NAME = System.getProperty("BLUE_ORGANIZATION_ROOT_FOLDER");
```

Comment:
conventionally, something like

```java
SystemProperties.getProperty(OrganizationFactoryImpl.class.getName() + ".rootFolderName");
```

Reply:
yup, but BO predates all of SystemProperties so all the other things require Java system properties, so going to leave this as is.